exporter/containerimage/export.go

```diff
@@ -36,13 +36,13 @@ import (
 )
 
 const (
-	keyPush           = "push"
-	keyPushByDigest   = "push-by-digest"
-	keyInsecure       = "registry.insecure"
-	keyUnpack         = "unpack"
-	keyDanglingPrefix = "dangling-name-prefix"
-	keyNameCanonical  = "name-canonical"
-	keyStore          = "store"
+	KeyPush           = "push"
+	KeyPushByDigest   = "push-by-digest"
+	KeyInsecure       = "registry.insecure"
+	KeyUnpack         = "unpack"
+	KeyDanglingPrefix = "dangling-name-prefix"
+	KeyNameCanonical  = "name-canonical"
+	KeyStore          = "store"
```

Comment:
As a follow-up, we should consider;

- making these strong-typed (if possible)
- documenting these (some of them may be clear, but still may require some knowledge (what does this mean?)

I know @jedevc has done some work on improving such consts in other areas (happy to do the work, but I may need some input on the docs side of things)